operator/api/v1alpha1/observatorium_types.go

```diff
@@ -178,11 +178,35 @@ type APITenant struct {
 	OIDC TenantOIDC `json:"oidc"`
 }
 
+type TLSSecret struct {
+	CertFile       string `json:"certFile"`
+	Name           string `json:"name"`
+	PrivateKeyFile string `json:"privateKeyFile"`
+	ReloadInterval string `json:"reloadInterval"`
+}
+
+type TLS struct {
+	TLSSecret TLSSecret `json:"secret"`
+}
+
+type MTLSConfigMap struct {
```

Comment:
We don't have any Observatorium-wide mTLS settings (anymore), so we should remove this struct